### PR TITLE
[bitnami/nats] Add `PersistentVolumeRetentionPolicy`

### PR DESCRIPTION
**Pull request by rblaine95**
### Description of the change

Added the `PersistentVolumeRetentionPolicy` to the NATS Statefulset so that PVCs can be configured to automatically be cleaned up when scaling or deleting the NATS Statefulset.
Disabled and set to Retain by default.

### Benefits

PVCs can be configured to automatically be deleted (retained by default) when scaling or deleting the statefulset.

### Possible drawbacks

None, disabled by default as in other charts (`bitnami/redis`, `bitnami/postgresql`, for example)

### Applicable issues

None

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/nats/values.yaml [hunk 1]
```diff
@@ -940,6 +940,19 @@ persistence:
   ##     app: my-app
   ##
   selector: {}
+## Persistent Volume Claim Retention Policy
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#persistentvolumeclaim-retention
+##
+persistentVolumeClaimRetentionPolicy:
+  ## @param persistentVolumeClaimRetentionPolicy.enabled Enable Persistent volume retention policy for postgresql Statefulset
```
Comment (juan131) [inline bitnami/nats/values.yaml:947]: Copy / paste typo:
```suggestion
  ## @param persistentVolumeClaimRetentionPolicy.enabled Enable Persistent volume retention policy for NATS statefulset
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please check my comments

File: bitnami/nats/values.yaml (hunk 1 above)
Comment (rblaine95) [reply bitnami/nats/values.yaml:947]: Ah, thank you so much for that catch!

**Review by juan131 — APPROVED** [review]
LGTM

**Comment by rblaine95** [comment]
Thank you very much @juan131 for the quick review! 🎉

**Pull request merged**